lib/core/lib/core/models/opportunity/timeline.tsp

```diff
@@ -33,14 +33,23 @@ model OppTimeline {
 /** Examples of the OppTimeline model */
 namespace Examples.Timeline {
   const opportunity = #{
-    appOpens: Fields.Examples.Event.openDate,
-    appDeadline: Fields.Examples.Event.deadline,
+    postDate: Fields.Examples.Event.postedDate,
+    closeDate: Fields.Examples.Event.closeDate,
     otherDates: #{
       anticipatedAward: #{
         name: "Anticipated award date",
+        eventType: EventType.singleDate,
         date: isoDate.fromISO("2025-03-15"),
         description: "When we expect to announce awards for this opportunity.",
       },
+      applicationPeriod: Fields.Examples.Event.applicationPeriod,
+      performancePeriod: Fields.Examples.Event.performancePeriod,
+      infoSessions: #{
+        name: "Info sessions",
+        eventType: EventType.other,
+        details: "Every other Tuesday",
+        description: "Info sessions for the opportunity",
+      },
```

Comment:
Here's an example of how each event type might be used collectively to describe an opportunity's timeline, with:

- Top-level posted and closed dates
- An application period
- A project period of performance for the grant itself
- Recurring info sessions